sdk/schemaregistry/azure-data-schemaregistry/src/test/java/com/azure/data/schemaregistry/SchemaRegistryClientTests.java

```diff
@@ -9,7 +9,7 @@
 import com.azure.core.exception.ResourceNotFoundException;
 import com.azure.core.http.policy.RetryPolicy;
 import com.azure.core.test.TestBase;
-import com.azure.data.schemaregistry.implementation.models.ServiceErrorResponseException;
+import com.azure.data.schemaregistry.implementation.models.ErrorException;
```

Comment:
We usually don't expose an `implementation.model` to the user?

Reply:
True. Let me update that to AzureException